apps/zotonic_launcher/priv/config/zotonic.config.in

```diff
@@ -330,7 +330,9 @@
       %%% Waiting period to delete files on the remote server after they are deleted locally
       %%% from the site. Possible values are "0" form an immediate delete, a number is a number
       %%% of seconds, or "N days", "N weeks" or "N months" (where N is a number).
-      {delete_interval, <<"0">>}
+      %%% Leave empty to use the default of mod_filestore, currently "5 weeks", which matches
+      %%% the number of weekly backups by mod_backup.
+      {delete_interval, ""}
```

Comment:
It should be empty list or empty binary?
```suggestion
      {delete_interval, <<>>}
```

Reply:
Both is ok - I settled for the empty string (list) as most other values in the config are strings (lists).